### PR TITLE
Document that parentheses are required for chained comparison operators (RFC 0).

### PR DESCRIPTION
**Pull request by ekse**

### CODE REVIEW

**Comment by ekse** [comment]
This is my first attempt at a contribution to Rust (yay!), let me know if something is wrong or if you would like the text worded differently.

File: src/expressions.md [hunk 1]
```diff
@@ -436,6 +436,9 @@ operators on standard types is given here.
   : Greater than or equal.
     Calls the `ge` method on the `std::cmp::PartialOrd` trait.
 
+Parentheses are required when chaining comparison operators. For example, the
+expression `a == b == c` is invalid and may be written as `((a == b) == c)`.
```
Comment (steveklabnik) [inline src/expressions.md]: could this be `(a == b) == c`, that is, without the outer parens? In `if` statements, those outer ones will cause a warning.

**Comment by steveklabnik** [comment]
One tiny comment, otherwise, this looks good! Thank you! 😄

File: src/expressions.md (hunk 1 above)
Comment (ekse) [inline src/expressions.md]: Good catch, it's fixed now.

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**